miden/src/cli/data.rs

```diff
@@ -20,7 +20,7 @@ use stdlib::StdLibrary;
 #[derive(Deserialize, Debug)]
 pub struct InputFile {
     pub stack_init: Vec<String>,
-    pub advice_tape: Option<Vec<String>>,
+    pub advice_stack: Option<Vec<String>>,
 }
```

Comment:
I wonder if we should rename `stack_init` to `operand_stack`. If yes, probably in another PR.

Reply:
Added `nice to have` to be either done or converted to future issue, before merging

Reply:
Agreed!

Reply:
Fixed on #785